packages/docs/docs/Usage/ec_rc_setup.md

```diff
@@ -1,11 +1,10 @@
-# Embedded Chat: A staple in excellent customer service
-
-An easy-to-use, full-stack component (React.js + backend behaviors) for embedding Rocket.Chat into your web app.
+---
+title: Quick Control Guide
+---
+# Quick Control Guide
 
 ![ec-demo-image](https://github.com/RocketChat/EmbeddedChat/assets/78961432/b85c7b8a-65e2-4a90-a843-f4072c942ac0)
 
-## EmbeddedChat RC App
-
 This monorepo hosts an RC app designed to enhance EmbeddedChat functionalities. It facilitates remote configuration of EmbeddedChat settings, implements secure login (storing the resume token as an HTTP-only cookie), and help integrates Rocket.Chat OAuth login capabilities.
```

Comment:
Remove this and add the following: 

EmbeddedChat provides 'Quick Control' to simplify remote configuration of its settings, including themes, via the RC app, which can be installed in your Rocket.Chat workspace. It also aids in implementing secure login by storing the resume token as an HTTP-only cookie and integrating Rocket.Chat's OAuth login capabilities.

Reply:
After installation, add a new heading titled ### Usage and include the following text:

Once the Quick Control RC app is installed, navigate to App Settings, select EmbeddedChat, and you can configure EmbeddedChat properties remotely and access the other mentioned features.

Reply:
done